pages/linux/slabtop.md

```diff
@@ -0,0 +1,29 @@
+# slabtop
+
+> Display kernel slab cache information in real time.
+> See also: `top`, `htop`, `atop`.
+> More information: <https://manned.org/slabtop>.
+
+- Start `slabtop`:
+
+`slabtop`
+
+- Sort by cache size:
+
+`slabtop {{[-s|--sort]}} {{cache_size}}`
+
+- Sort by number of objects:
+
+`slabtop {{[-s|--sort]}} {{number_objects}}`
+
+- Sort by object size:
+
+`slabtop {{[-s|--sort]}} {{size_object}}`
```

Comment:
```suggestion
`slabtop {{[-s|--sort]}} {{object_size}}`
```
Please update this placeholder to match naming conventions and ensure consistency across all examples.